### PR TITLE
Add Snyk badges to component and utility packages

### LINKED ISSUE #411: Add Snyk badge to each package
**Is your feature request related to a problem? Please describe.**
**Describe the solution you'd like**
**Describe the solution you'd like**

We are using Snyk to check for vulnerabilities in every package.json we should add the badge to show that each package is secure.

**Describe alternatives you've considered**
Not doing this, but it shows our commitment to security and excellent security of our packages.

**Testing notes**
[Tester to complete]

Dev insight: IMO No testing needed.

**Additional context**
n/a
- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/psammead/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by amywalkerdev**
Resolves #411 

**Overall change:**  Adds Snyk badges to each package readme to show vulnerabilities. 
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-figure/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+# psammead-figure - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-copyright%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-copyright%2Fpackage.json) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/figure--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-figure.svg)](https://www.npmjs.com/package/@bbc/psammead-figure) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
```
Comment (j-pendlebury) [inline packages/components/psammead-figure/README.md]: It's `psammead-figure` but links to `psammead-copyright`. Copy and paste? 😄

**Review by j-pendlebury — CHANGES_REQUESTED** [review]
just one comment but otherwise looks good

File: packages/components/psammead-figure/README.md (hunk 1 above)
Comment (amywalkerdev) [reply packages/components/psammead-figure/README.md]: Yes 😖

**Review by j-pendlebury — DISMISSED** [review]

**Review by dr3 — CHANGES_REQUESTED** [review]
All of these packages need to be bumped and have a changelog

**Comment by amywalkerdev** [comment]
> All of these packages need to be bumped and have a changelog

Done

**Review by j-pendlebury — DISMISSED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -8284,20 +8284,20 @@
       "dependencies": {
         "abbrev": {
           "version": "1.1.1",
-          "resolved": "",
+          "resolved": false,
```
Comment (dr3) [inline package-lock.json]: Can you reset this file to latest please 

Also might be worth checking your npm & node versions are matching the psammead ones

**Review by dr3 — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 2 above)
Comment (dr3) [reply package-lock.json]: Otherwise LGTM

Comment (amywalkerdev) [reply package-lock.json]: Done!

**Review by dr3 — APPROVED** [review]

**Review by dr3 — CHANGES_REQUESTED** [review]
Need to resolve conflicts

**Review by j-pendlebury — DISMISSED** [review]

**Review by dr3 — DISMISSED** [review]

**Comment by jamesbrumpton** [comment]
LGTM. Happy for this to be merged.

**Review by dr3 — APPROVED** [review]
merge merge merge merge

**Review by sareh — APPROVED** [review]

**Pull request merged**